WordPress/Classes/ViewRelated/Blog/My Site/Header/HomeSiteHeaderViewController+SiteActions.swift

```diff
@@ -106,17 +105,13 @@ extension SitePickerViewController {
 private enum MenuItem {
     case visitSite(_ handler: () -> Void)
     case shareSite(_ handler: () -> Void)
-    case addSite(_ handler: () -> Void)
```

Comment:
These are part of the "hidden" menu that you see on long-press on a site picker "chevron". "addSite" wasn't used. "switchSite" is already the primary action and makes no sense, so I removed it for simplicity.